x/concentrated-liquidity/math_test.go

```diff
@@ -7,30 +7,28 @@ import (
 )
 
 func (suite *KeeperTestSuite) TestGetLiquidityFromAmounts() {
-	testCases := []struct {
-		name              string
+	testCases := map[string]struct {
 		currentSqrtP      sdk.Dec
 		sqrtPHigh         sdk.Dec
 		sqrtPLow          sdk.Dec
 		amount0Desired    sdk.Int
 		amount1Desired    sdk.Int
 		expectedLiquidity string
 	}{
-		{
-			"happy path",
-			sdk.MustNewDecFromStr("70710678"),
-			sdk.MustNewDecFromStr("74161984"),
-			sdk.MustNewDecFromStr("67082039"),
-			sdk.NewInt(1000000),
-			sdk.NewInt(5000000000),
-			"1377.927096082029653542",
+		"happy path": {
+			currentSqrtP:      sdk.MustNewDecFromStr("70.710678118654752440"), // 5000
+			sqrtPHigh:         sdk.MustNewDecFromStr("74.161984870956629487"), // 5500
+			sqrtPLow:          sdk.MustNewDecFromStr("67.416615162732695594"), // 4545
+			amount0Desired:    sdk.NewInt(1000000),
+			amount1Desired:    sdk.NewInt(5000000000),
+			expectedLiquidity: "1517882343.751510418088349649",
 		},
 	}
 
-	for _, tc := range testCases {
+	for name, tc := range testCases {
 		tc := tc
 
-		suite.Run(tc.name, func() {
+		suite.Run(name, func() {
```

Comment:
Consider adding comments about what is happening inside `getLiquidityFromAmounts`.